crates/cxx-qt-build/src/lib.rs

```diff
@@ -498,6 +498,17 @@ impl CxxQtBuilder {
             qtbuild.version().major.to_string(),
         );
 
+        for minor in 0..qtbuild.version().minor {
+            let at_least_qt_version = format!(
```

Comment:
i guess one thing to note here is that it doesn't work across major versions.

Eg i can't do "at least 5.15", as it only loops through the minor version for the current major version.

I don't think that is a problem at the moment as we only care about 5.15+ for the 5.x series.

But i'm thinking here about if/when Qt 7 comes out then this is going to break ?

But to resolve this it'd need a manual loop ? as once we are eg 7.0 you can't loop over 6.x for the minor version.

Reply:
indeed. Perhaps we need to add a loop when major > 6
For the moment it's not necessary.
For qt7
We need to make range for qt6 = > create for qt 6.15 or other...
Otherwise we will not create at_least_qt_6_99 etc :)
=> We must improve it when we will know which will be the last minor qt6 version